### PR TITLE
Refer to timeliness in months

### PR DESCRIPTION
**Pull request by woodwoerk**
Turns out the actual donut segment labels come from the backend, so this just updates the donut description to use months when applicable.

image

### CODE REVIEW

File: locales/fi/actions.json [hunk 1]
```diff
@@ -52,7 +52,9 @@
   "actions-status": "Toimenpiteiden tila",
   "actions-status-help": "Aikataulussa olevien toimenpiteiden osuus",
   "actions-updated": "Aktiivisten toimenpiteiden päivitys",
-  "actions-updated-help": "Viimeisen {{count}} päivän aikana päivitettyjen toimenpiteiden osuus",
+  "actions-updated-help-days": "Viimeisen {{count}} päivän aikana päivitettyjen toimenpiteiden osuus",
+  "actions-updated-help-months_one": "Toimenpiteet, jotka on päivitetty viime kuussa",
```
Comment (woodwoerk) [inline locales/fi/actions.json:56]: @terotik can you suggest better copy for these?

Comment (terotik) [reply locales/fi/actions.json:56]: Maybe "viime kuussa" -> "viimeisen kuukauden aikana"

Comment (woodwoerk) [inline locales/fi/actions.json:56]:
```suggestion
  "actions-updated-help-months_one": "Toimenpiteet, jotka on päivitetty viimeisen kuukauden aikana",
```

Comment (woodwoerk) [reply locales/fi/actions.json]: Oh nice! I'll go with that next time

**Pull request merged**